conf/deployment/aws/upi_3az_rhel7_3m_3w.yaml

```
@@ -13,3 +13,6 @@ ENV_DATA:
   root_disk_size: 120
   root_disk: '/dev/sda1'
   availability_zone_count: 3
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-1499'
```

I think, that there should be `OCS-1449`: *OCS-1449 - OCS Deployment on: AWS + UPI + 3 AZ + RHEL 7 + 3 W nodes*
```suggestion
    deployment_id: 'OCS-1449'
```

test Case *OCS-1499* is *OCS Upgrade - Disconnected Upgrade (W/o internet) : on VMware UPI 1 AZ + CoreOS + VMFSDatastore + 3 W nodes*.

---

good catch

---

Fixed, good catch!